backbone.virtual-collection.js

```diff
@@ -102,6 +102,9 @@ var VirtualCollection = Backbone.VirtualCollection = Backbone.Collection.extend(
     this._indexAdd(model);
     this.listenTo(model, 'all', this._onAllEvent);
     this.trigger('add', model, this, options);  
+    if (this.comparator) {
```

Comment:
Make this an explicit undefined check to conform with other checks on this property in this file and to handle the possibility of a comparator on a property with a name that is an empty string.